### ISSUE TITLE
Document `attr` and `attribute`

### ISSUE THREAD

**Issue by hugovk** [python/devguide#1308]
Let's update https://devguide.python.org/documentation/markup/ to document that:

*  `.. attribute:: my-attr` should be used for defining attributes, and 
* ``` :attr:`my-attr` ``` used for referring to it.

**Comment by hugovk** [comment]
Yes, you can work on it :) We don't normally assign issues, and don't have a deadline, but someone else can have a go if this is still open after a few weeks.

Let me know if you've any questions.

Thank you!

**Comment by kokoavailable** [comment]
@hugovk 

I've submitted a PR.

https://github.com/python/devguide/pull/1311

Please let me know If there are any inaccuracies or misunderstanding in my request.

Thank you!
